client/components/takeQuiz/TakeQuiz.jsx

```diff
@@ -282,111 +396,253 @@ const TakeQuiz = ({ userId }) => {
       })
   };
 
+  const BlueRadio = withStyles({
+    root: {
+      color: "#303c6c",
+      "&$checked": {
+        color: "#303c6c"
+      }
+    },
+    checked: {}
+  })((props) => <Radio color="default" {...props} />);
+
   useEffect(() => {
     retrieveQuiz();
+    refreshFriends();
+    return () => {
+      setFriends([]);
+    };
   }, [])
```

Comment:
I think the [] at the bottom needs to be changed with user.id so it fires whenever a new user logs in

Reply:
Added a new useEffect that invokes refreshFriends() whenever [loggedInUser.id] changes!